apps/dashboard/package.json

```diff
@@ -121,6 +119,7 @@
     "express": "^4.21.0",
     "globals": "^15.9.0",
     "http-proxy-middleware": "^3.0.2",
+    "http-server": "^0.13.0",
```

Comment:
Is this still required?

Reply:
yes, to run the dashboard as a static build